azavea/temperate
# Remove default organization

## Overview
Most of the work appears to have been done in ecfcf56db0aeec58f9662209496e99f70deddf03, but this removes the User Organization organization from database along with a couple vestigial remnants. The original migration that added the default organization is left to avoid risk of undesirable interactions.

## Testing Instructions
- Install migrations
  - You shouldn't have an organization called "User Organization" in [the admin panel](http://localhost:8100/admin/users/planitorganization/)
  - Application should continue to work
- Run `docker-compose down` to destroy all docker containers and lose all data
- Run `./scripts/update` to recreate containers from scratch
  - Application should continue to work

Closes #598
